examples/shadowdom.js

```diff
@@ -0,0 +1,36 @@
+import { check } from 'k6';
+import { browser } from 'k6/x/browser';
+
+export const options = {
+  scenarios: {
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+            type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
+    checks: ["rate==1.0"]
+  }
+}
+
+export default async function() {
+  const page = browser.newPage();
+  page.setContent("<html><head><style></style></head><body>hello!</body></html>")
+  await page.evaluate(() => {
+    const shadowRoot = document.createElement('div');
+    shadowRoot.id = 'shadow-root';
+    shadowRoot.attachShadow({mode: 'open'});
+    shadowRoot.shadowRoot.innerHTML = '<p id="find">Shadow DOM</p>';
+    document.body.appendChild(shadowRoot);
+  });
+  const shadowEl = page.$("#find");
```

Comment:
Is there a reason why `$` (query) is used here instead of the `locator` API?

Reply:
Ah, I forgot that one while I was sorting things out. I'll fix it.

Reply:
https://github.com/grafana/xk6-browser/pull/1209/commits/75d33b3e871a6dcab74cc45135c15fd5b82483c7